packages/vuetify/src/components/VWindow/VWindow.ts

```diff
@@ -222,12 +222,17 @@ export default BaseItemGroup.extend({
       this.internalValue = this.getValue(item, lastIndex)
     },
     updateReverse (val: number, oldVal: number) {
-      const lastIndex = this.items.length - 1
-
-      if (val === lastIndex && oldVal === 0) {
-        return true
-      } else if (val === 0 && oldVal === lastIndex) {
-        return false
+      const itemsLength = this.items.length
```

Comment:
@nekosaur , please help to check if this change is compatible with fixing issue #11280 from this [PR](https://github.com/vuetifyjs/vuetify/pull/12357).

Thank you in advance :)